content/_releases/22/48.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Product Update - v22.48
+description: Details of the product update v22.48 from 1st of December 2022.
+layout: rel
+version: "22.48"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-12-01
+redirect_from:
+  - /releases/22.48.html
+---
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## OEM Feature
+
+Information in this section is intended for our customers who use the
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### White-labeled recipe management
+
+In this release we introduce the recipe management feature for creating and
+deploying the integration solutions in scale. You can now embed the entire
```

Comment:
at scale

Reply:
No need... -> Your clients can now access and deploy flows without leaving your platform or site - they simply select an integration that you list and enter their credentials.

Please note that you will need to trigger the necessary API calls to elastic.io so that the User, workspace, etc. are created and so that the flow is deployed.